changelog.txt

```diff
@@ -1,5 +1,6 @@
 v.Next
 ----------
+- [PATCH] Add YubiKit version variable to versions.gradle (#1689)
```

Comment:
This change should not need a changelog entry. We didn't patch anything here.

Reply:
my bad... I just removed this.